PowerShell.sublime-syntax

```diff
@@ -342,7 +343,7 @@ contexts:
           \-\w+?                                                               # Any "noun"
           (?:\.(?i:exe|cmd|bat|ps1))?\b                                        # More path stuff
         )
-      scope: support.function.powershell
```

Comment:
Adding `meta.function-call` results in all function calls being added to indexed symbol list.

As a result, you need to adjust syntax tests by adding "symbol tests" to all function calls.

Reply:
Unfortunately powershell tests mix scope and symbol tests, which may have quite some impact on changes like this.

Reply:
Got it, thank you.